pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -741,20 +739,22 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		networkXObject, err := getconfigCtx.pubNetworkXObjectConfig.Get(sysAdapter.NetworkUUID)
 		if err != nil {
 			// XXX when do we retry looking for the networkXObject?
-			errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s Network with UUID %s not found: %s",
+			errStr := fmt.Sprintf("Device Config Error. Port %s configured with "+
+				"UNKNOWN Network UUID (%s). Err: %s. Please fix the "+
+				"device configuration.",
 				port.IfName, sysAdapter.NetworkUUID, err)
-			log.Error(errStr)
-			port.ParseError = errStr
-			port.ParseErrorTime = time.Now()
+			log.Errorf("parseSystemAdapterConfig: %s", errStr)
+			port.SetErrorNow(errStr)
 		} else {
 			net := networkXObject.(types.NetworkXObjectConfig)
 			port.NetworkUUID = net.UUID
 			network = &net
 			if network.HasError() {
-				errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s Network error: %v",
-					port.IfName, network.Error)
-				port.ParseError = errStr
-				port.ParseErrorTime = network.ErrorTime
+				errStr := fmt.Sprintf("Port %s configured with Error Network. "+
```

Comment:
Thanks for making the error strings more usable for the admin.

But "with Error Network" doesn't read well. Should it say "with a network which has a (posted?) error. Network UUID: %s, Network Error: %s"?

Reply:
Done..	
	errStr := fmt.Sprintf("Port %s configured with a network "+
					"(UUID: %s) which has an error (%s).",
					port.IfName, port.NetworkUUID, network.Error)